server/server.js

```diff
@@ -21,35 +22,50 @@ app.use(bodyParser.urlencoded({
 app.use('/static', express.static(path.join(__dirname, '/static')));
 
 
+
+var bobList = [];
+
+
+function nowPlusXMinutes(numMinutes) {
+  return new Date(Date.now + numMinutes*60000);
+}
+
+
 // Main board
+
+// Define routes
 app.get('/', function (req, res) {
   res.sendFile(__dirname + '/templates/board.html');
 });
 
 // Send data to board
-const controller = require('./routes/controllerRoutes');
+const controller = require('./routes/controllerRoutes')(io, db);
 app.get('/controller', controller.GETindex);
 app.post('/controller', controller.POSTbob);
 app.get('/flavors', controller.GETflavors);
 app.get('/tags', controller.GETtags);
```

Comment:
Should there be `db` routes for querying databases? Or does everything work with DBs such that that would be redundant?